### PR TITLE
fixing issue #11

### PR DESCRIPTION
**Pull request by MaissaAlbaik**

### CODE REVIEW

**Comment by YazeedAlKhalaf** [comment]
please add to all `a` tags, the footer ones for example in the layou

**Comment by MaissaAlbaik** [comment]
Okay sure 

> please add to all `a` tags, the footer ones for example in the layou

File: src/components/ClassroomButton.astro [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ---
+
+
```
Comment (YazeedAlKhalaf) [inline src/components/ClassroomButton.astro:3]: remove this lines

**Review by YazeedAlKhalaf — APPROVED** [review]
LGTM

**Pull request merged**